xfce4-panel: Explicitly set panel size property

chaitu236 opened the pull request:
Explicitly set panel size as newer xfce4-panel versions require it.
Without this, the panel size defaults to 30 which isn't enough to fit
the existing panel items and also causes tasklist panel item to not
display correctly and respond to mouse inputs.

Azdo Bug: [1995530](https://ni.visualstudio.com/DevCentral/_workitems/edit/1995530)

### Testing

- Verified that a BSI built with these changes fixes tasklist plugin issue and panel size is same as in sumo.

--

sumo has a transparent panel that was set with 'background-alpha'
property. But this property doesn't work on hardknott anymore. So used
panel properties UI to get the new transparency settings. This also
updated the 'autohide' property to 'autohide-behavior'.

### Testing

- Panel is transparent with the changes like in sumo

![image](https://user-images.githubusercontent.com/8194523/172224707-be73b42b-ca5b-4a0e-8064-8e68e9487f33.png)

chaitu236 committed:
xfce4-panel: Explicitly set panel size property

Explicitly set panel size as newer xfce4-panel versions require it.
Without this, the panel size defaults to 30 which isn't enough to fit
the existing panel items and also causes tasklist panel item to not
display correctly and respond to mouse inputs.

Signed-off-by: Chaitanya Vadrevu <chaitanya.vadrevu@ni.com>

shruthi-ravi reviewed (approved):
Can you please describe the issue you are fixing in the commit message?

chaitu236 committed:
xfce4-panel: Make panel transparent

sumo has a transparent panel that was set with 'background-alpha'
property. But this property doesn't work on hardknott anymore. So used
panel properties UI to get the new transparency settings. This also
updated the 'autohide' property to 'autohide-behavior'.

Signed-off-by: Chaitanya Vadrevu <chaitanya.vadrevu@ni.com>

chaitu236 commented:
> Can you please describe the issue you are fixing in the commit message?
Done

chaitu236 commented:
Sorry for re-requesting review as I pushed another commit to fix transparency.
